src/main/java/com/conveyal/datatools/manager/jobs/DeployJob.java

```diff
@@ -569,24 +578,31 @@ private void replaceEC2Servers() {
 
                 // Check if a new image of the instance with the completed graph build should be created.
                 if (otpServer.ec2Info.recreateBuildImage) {
```

Comment:
(Couldn't ref this line, so adding code ref below):

Just stumbled across this uncommented line. Do you recall what this persistence operation is intended to do? If so, could you add a comment?

https://github.com/ibi-group/datatools-server/blob/06138de0bfc3abd6afa86c74e597a2c8fe70c6b6/src/main/java/com/conveyal/datatools/manager/jobs/DeployJob.java#L577

Reply:
This doesn't seem to be needed anymore, so I removed it.